sklearn/iris_training.py

```diff
@@ -32,27 +31,30 @@
 
 
 # [START load-into-pandas]
-# Load data into pandas
+# Load data into pandas, then use `.values` to get NumPy arrays
 iris_data = pd.read_csv(iris_data_filename).values
 iris_target = pd.read_csv(iris_target_filename).values
+
+# Convert one-column 2D array into 1D array for use with scikit-learn
 iris_target = iris_target.reshape((iris_target.size,))
 # [END load-into-pandas]
 
 
 # [START train-and-save-model]
 # Train the model
-classifier = svm.SVC(verbose=True)
+classifier = svm.SVC(gamma='auto', verbose=True)
 classifier.fit(iris_data, iris_target)
 
 # Export the classifier to a file
-model = 'model.joblib'
-joblib.dump(classifier, model)
+model_filename = 'model.joblib'
```

Comment:
Can we make this a constant at the top 
```
MODEL_FILENAME = 'model.joblib'
```

Reply:
That makes sense for organization, but because this bit of code is getting included in the documentation [next to the model file naming requirements](https://cloud.google.com/ml-engine/docs/scikit/getting-started-training#train_and_save_a_model) I think it integrates best with the tutorial by leaving it where it is. What do you thinK?

Reply:
LGTM